src/style.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --primary-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+  --light-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h3,
+h4 {
+  margin: 0;
+  font-weight: 400;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 0 auto 40px;
+  display: block;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.card__feedback {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__stars :nth-child(5) {
+  background-image: url("./images/star.svg");
+}
+
```

Comment:
[checklist p11](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

You need to change your project structure, you can create multiple .css files for each BEM block.